nix/overlays/nixpkgs-overrides.nix

```diff
@@ -1,7 +1,9 @@
 self: super: {
+  naersk = super.callPackage ((import ../sources.nix).naersk) { };
```

Comment:
Ah I worked it out, it's needed for the build of `nixpkgs-fmt`, and it's all overlay magic!